pkg/engine/detectors.go

```diff
@@ -0,0 +1,42 @@
+package engine
+
+import (
+	"strings"
+
+	"github.com/trufflesecurity/trufflehog/v3/pkg/context"
+	"github.com/trufflesecurity/trufflehog/v3/pkg/detectors"
+)
+
+// Detectors only returns a specific set of detectors if they are specified in the
+// detectors list and are valid. Otherwise, it returns the default set of detectors.
```

Comment:
__strong suggestion__: I think including a detector in this list that ends up being invalid should be a fatal error. Otherwise a user will do a (potentially very long) scan, and not necessarily know that they didn't actually scan for their githib token or some other typo or misnamed detector.

Reply:
done ✅

Reply:
Would it be simple enough to include the misconfigured detectors by putting them in a separate list output from `filterDetectors` or something like that? Just so we can tell the user specifically which ones aren't valid?